playbook/app/pb_kits/playbook/pb_tooltip/docs/_tooltip_position.jsx

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+import React from 'react'
+
+import { Body, Button, Flex, Tooltip } from '../..';
+
+const TooltipPosition = (props) => {
+  return (
+    <Flex 
+        gap='md'
+        justify='center'
+    >
+      <div style={{
+        overflow: 'scroll',
+        height: '85px',
+        width: '85px'
+      }}>
+        <Body text='Scroll down this div and hover the button' />
```

Comment:
Code looks great! Just a small thing, the text inside the Body kit here is not visible in dark mode, could you make that fix so everything looks good in dark mode as well?
![Screenshot 2023-03-29 at 10 38 30 AM](https://user-images.githubusercontent.com/73710701/228574615-b0b34c96-e7dd-4500-985c-8465bb7893ff.png)

Reply:
Hey Nida, thanks for your feedback. I spread the props in the `body` kit and now it is displaying in dark mode. Could you check again?